src/api-reference/invoice/purchase-request-endpoints.markdown

```diff
@@ -215,23 +224,30 @@ None
 |`comments`|`string`|-|Internal comments you want to record related to this record
 |`custom1 through custom20`|`string`|-|Each custom field used should have its own row in the message. If the field is tied to a connected list, the accepted value is the Item Code setup for the list in Concur
```

Comment:
Concur Invoice instead of Concur

Reply:
List is a common API in Concur and not specific to Concur Invoice. They can use this list data in another concur data as well. Please let me know.

Reply:
Fixed in fa1772b9